pyfolio/pos.py

```diff
@@ -171,3 +173,35 @@ def get_turnover(transactions, positions, period=None):
     turnover = traded_value / 2.0
     turnover_rate = turnover / portfolio_value
     return turnover_rate
+
+
+def get_sector_exposures(positions, symbol_sector_map):
+    """
+    Sum position exposures by sector.
+
+    Parameters
+    ----------
+    positions : pd.DataFrame
+        Contains position values or amounts.
+    symbol_sector_map : dict
+        Sector to security identifier mapping.
+        Sectors as keys, security ids as values.
+
+    Returns
+    -------
+    positions_alloc : pd.DataFrame
+        Sectors and their allocations.
+    """
+    unmapped_pos = np.setdiff1d(positions.drop('cash', axis=1).columns.values,
```

Comment:
You drop `cash` twice in this function.  You probably want to just do that once and then save the result.